src/Views/WifiPage.vala

```diff
@@ -531,8 +532,24 @@ namespace Network {
                 wifi_dialog.run ();
                 wifi_dialog.destroy ();
             } else {
+                if (NM.@80211ApSecurityFlags.KEY_MGMT_OWE in row.ap.get_rsn_flags () ||
```

Comment:
This almost looks like it is replacing a different connection type? Is that right/a good idea?

Reply:
From what I understand, it's not replacing, just adding the required properties (uuid and algorithm "owe") to the blank new connection before making the connection without the wifi dialog, if the owe mode is detected which doesn't have a password.

Similar to what is being done for WPA connections except there a wifi dialog is being shown to ask for the password.

Reply:
I see. I think this being in the else block after the is_secure check is making think this section was for open wifi APs. If that isn't the case *or* it is but this doesn't affect connecting to open networks, then I think this code seems alright. I sadly have no way to actually test though :(